src/Symfony/Component/Cache/Tests/Adapter/CouchbaseBucketAdapterTest.php

```diff
@@ -50,8 +50,8 @@ public function createCachePool($defaultLifetime = 0): CacheItemPoolInterface
                 .':'.getenv('COUCHBASE_PASS')
                 .'@'.getenv('COUCHBASE_HOST')
                 .'/cache')
-            : self::$client;
+            : $this->client;
 
-        return new CouchbaseBucketAdapter($client, str_replace('\\', '.', __CLASS__), $defaultLifetime);
+        return new \Symfony\Component\Cache\Adapter\CouchbaseBucketAdapter($client, str_replace('\\', '.', __CLASS__), $defaultLifetime);
```

Comment:
Changing from an import to FQCN allows to add `trigger_deprecation()` and don't have the self-deprecation. Indeed, tests from this class can be skipped which leads to not checking the raised deprecation. Now, the deprecation is checked whenever the tests are run.

Reply:
The tests of the deprecated adapter should get `@group legacy`

Reply:
Oh of course, totally forgot about it 🤦 I removed the FQCN and revert to the import. Thank you 🙏